Create Docs

Mineinjava opened the pull request:
I created the docs. These docs *should* be able to run on ReadTheDocs.

TODO:

- [ ] add more docstrings

---
Made with `sphinx.ext.autodoc`.

All docstrings must use ReStructuredText format.

Mineinjava committed:
Create Docs

I created the docs. These docs *should* be able to run on ReadTheDocs.

TODO:

- [ ] add more docstrings

---
Made with `sphinx.ext.autodoc`. All docstrings must use ReStructuredText format.

LennyPhoenix reviewed (approved):
Wow! Nice work on getting all that set up, should I merge this in now or do you want to add docstrings first?

LennyPhoenix commented on docs/discordsdk.rst:
```diff
@@ -0,0 +1,157 @@
+   :show-inheritance:
+
+discordsdk.event module
+-----------------------
```
This module is only used internally, so we might not need to put this on the auto-docs.

Mineinjava replied:
done!

Mineinjava committed:
remove discordsdk.event module

Removed the `discordsdk.event` module from autodoc as per [this review](https://github.com/LennyPhoenix/py-discord-sdk/pull/6#discussion_r579552048)

Mineinjava commented:
Lets merge this before adding docstrings.
